pyproject.toml

```diff
@@ -1,41 +1,25 @@
 [build-system]
-requires = ["setuptools >= 75.3.0", "wheel"]
+requires = ["setuptools >= 72.1.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-include-package-data = false
-
-[tool.setuptools.packages.find]
-include = ["yoyodyne*"]
-exclude = ["examples*"]
-
 [project]
 name = "yoyodyne"
-version = "0.3.3"
+version = "0.4.0"
```

Comment:
I think this can be 1 beta at this stage no?

Reply:
You're thinking like I am; I will advance it to that as soon as I have #5 (good tests) going. I know how to integration-test this from previous work with LightningCLI.